packages/dev/serializers/src/glTF/2.0/glTFUtilities.ts

```diff
@@ -1,137 +1,455 @@
-import type { IBufferView, AccessorComponentType, IAccessor } from "babylonjs-gltf2interface";
-import { AccessorType } from "babylonjs-gltf2interface";
+/* eslint-disable jsdoc/require-jsdoc */
 
-import type { FloatArray, Nullable } from "core/types";
+import type { IBufferView, AccessorComponentType, IAccessor, INode } from "babylonjs-gltf2interface";
+import { AccessorType, MeshPrimitiveMode } from "babylonjs-gltf2interface";
+
+import type { DataArray, IndicesArray, Nullable } from "core/types";
 import type { Vector4 } from "core/Maths/math.vector";
-import { Vector3 } from "core/Maths/math.vector";
+import { Quaternion, TmpVectors, Matrix, Vector3 } from "core/Maths/math.vector";
+import { VertexBuffer } from "core/Buffers/buffer";
+import { Material } from "core/Materials/material";
+import { TransformNode } from "core/Meshes/transformNode";
+import { Mesh } from "core/Meshes/mesh";
+import { InstancedMesh } from "core/Meshes/instancedMesh";
+import { enumerateFloatValues } from "core/Buffers/bufferUtils";
+import type { Node } from "core/node";
+
+// Matrix that converts handedness on the X-axis.
+const convertHandednessMatrix = Matrix.Compose(new Vector3(-1, 1, 1), Quaternion.Identity(), Vector3.Zero());
+
+// 180 degrees rotation in Y.
+const rotation180Y = new Quaternion(0, 1, 0, 0);
 
 /**
- * @internal
+ * Creates a buffer view based on the supplied arguments
+ * @param bufferIndex index value of the specified buffer
+ * @param byteOffset byte offset value
+ * @param byteLength byte length of the bufferView
+ * @param byteStride byte distance between conequential elements
+ * @returns bufferView for glTF
  */
-export class _GLTFUtilities {
-    /**
-     * Creates a buffer view based on the supplied arguments
-     * @param bufferIndex index value of the specified buffer
-     * @param byteOffset byte offset value
-     * @param byteLength byte length of the bufferView
-     * @param byteStride byte distance between conequential elements
-     * @param name name of the buffer view
-     * @returns bufferView for glTF
-     */
-    public static _CreateBufferView(bufferIndex: number, byteOffset: number, byteLength: number, byteStride?: number, name?: string): IBufferView {
-        const bufferview: IBufferView = { buffer: bufferIndex, byteLength: byteLength };
-        if (byteOffset) {
-            bufferview.byteOffset = byteOffset;
-        }
-        if (name) {
-            bufferview.name = name;
-        }
-        if (byteStride) {
-            bufferview.byteStride = byteStride;
-        }
+export function createBufferView(bufferIndex: number, byteOffset: number, byteLength: number, byteStride?: number): IBufferView {

```

Comment:
As noted in the other comment, we are settling on PascalCase for module level functions.

Reply:
I've gone ahead and updated all exported module-level functions to PascalCase in the PR. Done https://github.com/SergioRZMasson/Babylon.js/pull/12/commits/675312956f11e6b222221d093f5aaf6fb20fec6c